### DISCUSSION TITLE
page-key attribute of NuxtPage is matched - how to exclude attributes from matching?

### DISCUSSION THREAD

**Discussion by samydoesit** [unocss/unocss#1731] [Q&A] [answered]
Hello is this correct behavior when unocss generates a class based on the page-key attribute of NuxtPage's?
<img width="1298" alt="image" src="https://user-images.githubusercontent.com/15785987/195299551-b1464d88-0bd4-459f-96f9-ad71df3c0aee.png">
https://stackblitz.com/edit/github-ncxynv-q4oa7f?file=app.vue

Is there a way to control this behavior without prefixing?

**Comment by zyyv** [comment, marked as answer]
UnoCSS does its best to parse tokens and generate css. 
If you don't use it elsewhere, you can use `blocklist`.

https://github.com/unocss/unocss/blob/477ce2b5339e780dd2ed2611714149944a77caa7/packages/core/src/types.ts#L322-L326

[Playground](https://uno.antfu.me/play/?html=DwEwlgbgBADghgcwKYFoDWSCeBeARAZwBc5CwBjXKAegD4AoIA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbhwod6gBccADawRrCbFQwagC0fraoGmRIorbQQRqUyBqKALq5JN5wVAAW0DARMFRBXj4%2B4ZHRIDGl5ZUaGTCmMDG2RHBdJj3QVlhEqHBlAG6oUAGDPV1WtklFCoU%2B-FLVIUWbkrSyEO4AlBt1W7QMTKwcXNyn5-sCMCJinnt1VMLLqEEAjAA6ABMTzqwmQWAyJUYYGqAHp4dQQIDSvDVnVFGcigUiiw0sIANa2YBRIJhKJWTjCDS4hQnAxAA&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g)
